pkg/tracing/plugin/otlp.go

```diff
@@ -35,48 +37,77 @@ import (
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
 	"go.opentelemetry.io/otel/propagation"
-	"go.opentelemetry.io/otel/sdk/resource"
 	"go.opentelemetry.io/otel/sdk/trace"
-	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
 )
 
 const exporterPlugin = "otlp"
 
+// OTEL and OTLP standard env vars
+// See https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/
+const (
+	sdkDisabledEnv = "OTEL_SDK_DISABLED"
+
+	otlpEndpointEnv       = "OTEL_EXPORTER_OTLP_ENDPOINT"
+	otlpTracesEndpointEnv = "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT"
+	otlpProtocolEnv       = "OTEL_EXPORTER_OTLP_PROTOCOL"
+	otlpTracesProtocolEnv = "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
+
+	otelTracesExporterEnv = "OTEL_TRACES_EXPORTER"
+	otelServiceNameEnv    = "OTEL_SERVICE_NAME"
+)
+
 func init() {
 	registry.Register(&plugin.Registration{
 		ID:     exporterPlugin,
 		Type:   plugins.TracingProcessorPlugin,
 		Config: &OTLPConfig{},
 		InitFn: func(ic *plugin.InitContext) (interface{}, error) {
-			cfg := ic.Config.(*OTLPConfig)
-			exp, err := newExporter(ic.Context, cfg)
+			if err := warnOTLPConfig(ic); err != nil {
+				return nil, err
+			}
+			if err := checkDisabled(); err != nil {
```

Comment:
Scenario 1:
None of the OTEL variables are set

Scenario 2:
Only OTEL_SDK_DISABLED is set and it is set to `0`

Do we treat these differently? same?

Reply:
Same. I suppose we could allow the defaults through if OTEL_SDK_DISABLED is set to 0.

Reply:
thanks